### PR TITLE
New features added!

### PR DESCRIPTION
**Pull request by ZobairQ**
I have added new features!
You can now select and de-select all files with a button rather than right clicking.
You can now open your images in Lightroom with a button.
You can now open your images in photoshop ( right click -> edit in -> Photoshop)
If no image is selected, the delete button is disabled!

I have added [CallerMemberName] attribute to NotifyPropertyChanged for improvement.
screenshot_1
screenshot_2
screenshot_3
screenshot_4

### CODE REVIEW

File: CameraControl.Core/Classes/PhotoSession.cs [hunk 1]
```diff
@@ -780,6 +783,117 @@ public void SelectLiked()
                 }
             }
         }
+        /// <summary>
+        /// This method uses the System.Diagnostics.ProcessStartInfo to import selected pictures into lightroom
+        /// </summary>
+        public void OpenInLightroom()
+        {
+            System.Diagnostics.ProcessStartInfo lr = new System.Diagnostics.ProcessStartInfo();
+            string path = GetProperLightroomPath( GetPathForExe("Photoshop.exe"));
```
Comment (dukus) [inline CameraControl.Core/Classes/PhotoSession.cs]: What if the Photoshop is not installed just the lightroom ?

**Comment by dukus** [comment]
Can you set the visibility of the buttons and the menu based on lightroo/Photoshop is installed or not ?

**Comment by dukus** [comment]
If no images is selected the delete button delete the current image so should be enabled even no image is selected

**Comment by ZobairQ** [comment]
The can execute method should do that already.  I do check the registry for installed apps

**Comment by dukus** [comment]
Yes, but if i know right only disable the button, not hide the button

**Comment by ZobairQ** [comment]
Yes which does not make any sense + it will result in accidental deletes.  I Wonder why you built it in that way?

**Comment by ZobairQ** [comment]
You want it to hide it completely?

**Comment by ZobairQ** [comment]
By the way,  i wanted to talk to you about a matter in the code.  1. What is the reason for using mvvm tool Kit?  Even though you have a generic relaycommand?

File: CameraControl.Core/Classes/PhotoSession.cs (hunk 1 above)
Comment (ZobairQ) [inline CameraControl.Core/Classes/PhotoSession.cs]: I see,  I did not think of a case like it.  I will fix it ASAP

**Comment by dukus** [comment]
For delete : the user is asked if he/she want to continue and the file is moved in recycled bin.  If you wan't to delete a file is somehow annoying to select it first by mouse

The relaycommand should be removed when i started to use mvvm light but lack of free time i forgot to remove it. For me seems more easy some helper library to be maintained by somebody else,

**Comment by ZobairQ** [comment]
For the delete i understand your concern,  however,  what hoppens if no file is available?  However it is no big deal.  

I am not sure about mvvm light though,  i find it more of disadvantage to make use of exactly that library.  I might be outdated but last time I checked  they had many things not working properly.

**Pull request merged**